packages/docs/getting-started/changelog.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+- Deprecate :visited link styling
```

Comment:
This is shipping, why put in unreleased section?

Reply:
We don't have a section for `0.4.0` yet, and I didn't want any confusion before we had a release branch ready.

We should prolly standardize on this so that every PR includes the associated changelog update. Once we start planning toward a release, that might be easier.